modules/@apostrophecms/schema/ui/apos/logic/AposInputSlug.js

```diff
@@ -14,7 +14,7 @@ export default {
     return {
       conflict: false,
       isArchived: null,
-      originalSlugPartsLength: null
+      originalParentSlug: ''
```

Comment:
We don't check the slug length but we check if the parent slug has changed. if yes we don't update slug.